assets/src/components/routeLadder.tsx

```diff
@@ -66,14 +70,51 @@ export const Header = ({
         </Tippy>
       )}
       <div className="c-route-ladder__close-button-container">
-        <CloseButton closeButtonType="l_darker" onClick={onClose} />
+        <OldCloseButton closeButtonType="l_darker" onClick={onClose} />
       </div>
 
       <div className="c-route-ladder__route-name">{routeName}</div>
     </div>
   )
 }
 
+// TODO: delete old header after roll-out
+export const NewHeader = ({
+  routeName,
+  onClose,
+  hasAlert,
+}: {
+  routeName: string
+  onClose: () => void
+  hasAlert: boolean
+}) => {
+  return (
+    <div className="c-new-route-ladder__header">
```

Comment:
hmm, Figma says this is supposed to be a bootstrap Card, is there a reason to prefer custom CSS for this instead of bootstrap classes?

Reply:
What if you instead wrap the SVG in a `div` under the `Tippy`?

Reply:
Agh, there were a couple things I missed with this Figma handoff. I'm revisiting now. Thank you!